### ISSUE TITLE
Hide form fields based on value of other form fields?

### ISSUE THREAD

**Issue by joshuakoh1** [danvick/flutter_form_builder#316]
Using _fbKey.currentState.fields['dependent_field'].currentState.value does not work, throws the error that customState was called on null

**Comment by Dennis-Mwea** [comment]
You could use `Visibility()` widget and set `visible` to either true or false depending on the value of the other field.

**Comment by joshuakoh1** [comment]
@Dennis-Mwea Do you mean I'll declare a separate bool value on init then set that with onChanged of the form field that is responsible for the visibility value? Because my current setup throws an error "NoSuchMethodError: The method '[]' was called on null" as the state has not been initialised:
```
 Visibility(
                            visible: this._fbKey.currentState?.fields['dependent_field']?.currentState.value,
                            maintainState: true,
                            child: Text('Hello'),
                          ),
```

**Comment by Dennis-Mwea** [comment]
@joshuakoh1 Yeah you have to declare separate bool value on init then set that with onChanged of the form field that is responsible for the visibility value.

**Comment by deandreamatias** [comment]
Hi @joshuakoh1!
This still an issue?

**Comment by deandreamatias** [comment]
Due to lack of response and an old error, I will close this issue.
If the bug still exists, feel free to open a [new issue](https://github.com/danvick/flutter_form_builder/issues/new/choose)
